src/funding/funding.ts

```diff
@@ -338,6 +339,7 @@ export class Funding {
    */
   private setTokenContractInfo(token: ITokenFunding, dao: IDAO): void {
     //get the additional token information from the contract for this token
+    console.log("Dao token transactions", this.deal.daoTokenTransactions.get(dao));
```

Comment:
will need to delete this eventually

Reply:
I'll make sure to delete this in the next PR, thanks!